### PR TITLE
Feature/civ 1267 do you live with a partner

### PR DESCRIPTION
**Pull request by GorilaNaranja**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-1267

### Change description ###
View 'Do you live with a partner' screen

**Does this PR introduce a breaking change?** (check one with "x")

```
[X] Yes
[ ] No
```

### CODE REVIEW

**Review by sampankumar — APPROVED** [review]

**Review by laarmada — APPROVED** [review]
ok

**Review by LeonardoPalmeiro — APPROVED** [review]

File: src/main/modules/statementOfMeans/partner/cohabitingService.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import {Cohabiting} from '../../../common/form/models/statementOfMeans/partner/cohabiting';
+import {getDraftClaimFromStore, saveDraftClaim} from '../../draft-store/draftStoreService';
+import {StatementOfMeans} from '../../../common/models/statementOfMeans';
+
+const {Logger} = require('@hmcts/nodejs-logging');
+const logger = Logger.getLogger('disabilityService');
```
Comment (osheaf01) [inline src/main/modules/statementOfMeans/partner/cohabitingService.ts:6]: 'cohabitingService'? Copy/paste ;)

File: src/main/modules/statementOfMeans/partner/cohabitingService.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+export class CohabitingService {
+
+  public async getCohabiting(claimId: string) {
+    try {
+      const civilClaimResponse = await getDraftClaimFromStore(claimId);
+      logger.info(civilClaimResponse);
+      if (civilClaimResponse && civilClaimResponse.case_data && civilClaimResponse.case_data.statementOfMeans && civilClaimResponse.case_data.statementOfMeans.disability) {
+        return civilClaimResponse.case_data.statementOfMeans.disability;
+      }
```
Comment (osheaf01) [inline src/main/modules/statementOfMeans/partner/cohabitingService.ts:16]: Did you mean civilClaimResponse.case_data.statementOfMeans.cohabiting?

File: src/main/routes/features/response/statementOfMeans/partner/partnerController.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    const validator = new Validator();
+    const errors: ValidationError[] = validator.validateSync(cohabiting);
+    if (errors && errors.length > 0) {
+      cohabiting.errors = errors;
+      renderView(cohabiting, res);
+    } else {
+      cohabitingService.saveCohabiting(req.params.id, cohabiting);
+      if (cohabiting.option == 'yes') {
+        res.redirect(PARTNER_AGE_URL);
+      } else {
```
Comment (osheaf01) [inline src/main/routes/features/response/statementOfMeans/partner/partnerController.ts:34]: If cohabiting.option is not 'yes', is it certain it's always 'no'? Log an error if it's neither?

File: src/main/modules/statementOfMeans/partner/cohabitingService.ts (hunk 2 above)
Comment (laarmada) [reply src/main/modules/statementOfMeans/partner/cohabitingService.ts:16]: thanks @osheaf01 !!!

Comment (GorilaNaranja) [reply src/main/modules/statementOfMeans/partner/cohabitingService.ts:16]: Done

File: src/main/routes/features/response/statementOfMeans/partner/partnerController.ts (hunk 3 above)
Comment (GorilaNaranja) [reply src/main/routes/features/response/statementOfMeans/partner/partnerController.ts:34]: We are checking for errors before, so I think if we arrive to this part it is sure that the option is "yes" or "no", right?

**Pull request merged**